.github/workflows/wpt-nightly.yml

```diff
@@ -0,0 +1,149 @@
+name: Synchronize WPT Nightly
+
+on:
+  pull_request:
```

Comment:
```suggestion
```

Reply:
We don't want to run this on every pull request.

Reply:
Fixed in 1b9bda0.